src/getters/crunchbase.py

```diff
@@ -0,0 +1,133 @@
+"""
+discovery_utils.src.getters.crunchbase.py
+
+Module for easy access to downloaded CB data on S3.
+
+"""
+import pandas as pd
+import boto3
+import os
+from dotenv import load_dotenv
+from botocore.client import BaseClient
+from src.utils import s3 as s3
+from src.utils import timestamps as ts
+from datetime import datetime, timedelta
+import logging
+import json
+
+# logging level
+logging.basicConfig(level=logging.INFO)
+
+load_dotenv()
+
+# Retrieve AWS file information from environment variables
+BUCKET_NAME_RAW = os.getenv("BUCKET_NAME_RAW")
+FILE_NAMES_RAW = json.loads(os.getenv("FILE_NAMES_RAW", '[]'))
```

Comment:
I used this to load in the list from environment variables (which I think needs to be loaded as a string). I had some issues with this, but this seemed to resolve it. Would be good to know your preferred way of dealing with "List" type environment variables.

Reply:
You could pass in file names as comma separated values, with or without spaces, then do a string `.strip().split(",")`

But this works either way